TerrorConsole/Assets/_Main/Scripts/Shadow/Shadow_tutorial.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class Shadow_tutorial : MonoBehaviour
```

Comment:
Thi script.... has like a million problems :'D
Ask @FerchusGames for a deep dive into how to fix it cause there are a lot of things.

In general this script should only have a method that starts the animation when the door is opened, that can be achieved with adding a `UnityEvent` to the door object and the method on this script via serialization.